Testing/integration/VM/setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -xeuo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+read -p "About to run visudo, set %admin ALL = (ALL) NOPASSWD: ALL"
+sudo visudo
+
+# Unpack and install xcode
+if [ ! -e /Applications/Xcode.app ]; then
+	(cd /Applications && xip --expand ${SCRIPT_DIR}/Xcode*.xip)
+	sudo xcode-select -s /Applications/Xcode.app
+	sudo xcodebuild -license accept
+	# ... and open it to fixup stuff
+	open /Applications/Xcode.app
+fi
+
+# Install brew
+if ! command -v brew &> /dev/null; then
+	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+	echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> $HOME/.zprofile
+	echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> $HOME/.bashrc
+	eval "$(/opt/homebrew/bin/brew shellenv)"
+fi
+
+# Install things from brew
+if ! command -v go &> /dev/null; then
+	brew install bazelisk go jq
+fi
+
+# Install rosetta (for test binaries)
+softwareupdate --install-rosetta --agree-to-license
+
+# Install actions runner
+mkdir ~/actions-runner
+pushd ~/actions-runner
+curl -o actions-runner-osx-arm64-2.296.0.tar.gz -L https://github.com/actions/runner/releases/download/v2.296.0/actions-runner-osx-arm64-2.296.0.tar.gz
+tar xzf ./actions-runner-osx-arm64-2.296.0.tar.gz
+./config.sh --url https://github.com/kallsyms/santa
```

Comment:
Is this correct? Why not use the main repo?

Reply:
Went ahead and changed - this was part of the first bulleted thing to do before merging.